global/concourse/templates/daemonset.yaml

```diff
@@ -0,0 +1,139 @@
+{{ range .Values.teams }}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ $.Values.worker.name }}-{{ .name }}
+  labels:
+    app: {{ $.Values.worker.name }}-{{ .name }}
+    chart: "{{ $.Chart.Name }}-{{ $.Chart.Version }}"
+    release: "{{ $.Release.Name }}"
+spec:
+  updateStrategy:
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      app: {{ $.Values.worker.name }}-{{ .name }}
+  template:
+    metadata:
+      labels:
+        app: {{ $.Values.worker.name }}-{{ .name }}
+        release: "{{ $.Release.Name }}"
+    spec:
+      nodeSelector:
+        ccloud.sap.com/nodepool: {{ .nodePool | quote }}
+      tolerations:
+      - key: "team"
+        operator: Equal
+        value: {{ .name | quote }}
+        effect: "NoSchedule"
+      serviceAccountName: concourse-worker
+      initContainers:
+      - name: {{ $.Values.worker.name }}-{{ .name }}-init-rm
+        image: "{{ $.Values.concourse.image }}:{{ $.Values.concourse.imageTag }}"
+        imagePullPolicy: {{ $.Values.concourse.imagePullPolicy | quote }}
+        command:
+        - /bin/sh
+        args:
+        - -ce
+        - |-
+          rm -rf {{ $.Values.concourse.concourse.worker.workDir }}/*
+        volumeMounts:
+        - name: concourse-work-dir
+          mountPath: {{ $.Values.concourse.concourse.worker.workDir | quote }}
+      containers:
+      - name: {{ $.Values.worker.name }}-{{ .name }}
+        image: "{{ $.Values.concourse.image }}:{{ $.Values.concourse.imageTag }}"
+        imagePullPolicy: {{ $.Values.concourse.imagePullPolicy | quote }}
+        command:
+        - /bin/sh
+        args:
+        - -ce
+        - |-
+          update-ca-certificates &&
+          /usr/local/concourse/bin/concourse worker
+        livenessProbe:
+          failureThreshold: 5
+          initialDelaySeconds: 10
+          periodSeconds: 15
+          timeoutSeconds: 3
+          httpGet:
+            path: /
+            port: 8888
+        lifecycle:
+          preStop:
+            exec:
+              command:
+              - "kill"
+              - "-s"
+              - "SIGUSR2"
+              - "1"
+        env:
```

Comment:
We should add the following setting, ideally configureable by value:
```
      CONCOURSE_AUTH_DURATION:                 72h
      CONCOURSE_DEFAULT_BUILD_LOGS_TO_RETAIN:  50
      CONCOURSE_MAX_BUILD_LOGS_TO_RETAIN:      100
```

Reply:
This is set via values in official chart web daemonset.